apps/workflows-dashboard/src/main.tsx

```diff
@@ -4,6 +4,7 @@ import * as ReactDOM from 'react-dom/client';
 import { RouterProvider } from 'react-router-dom';
 import './index.css';
 import { initializeMonitoring } from '@/initialize-monitoring/initialize-monitoring';
+import '../public/config.js?url';
```

Comment:
if you add it in here it would take in place just if you build the app again

Reply:
Yes it will